tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -1271,12 +1270,6 @@ public bool DiagnosticSourceEnabled
         /// </summary>
         internal bool IsRunningInAzureFunctions => AzureAppServiceMetadata?.IsFunctionsApp ?? false;
 
-        /// <summary>
-        /// Gets a value indicating whether the tracer is running in an Azure Function on a
-        /// consumption plan
-        /// </summary>
-        internal bool IsRunningMiniAgentInAzureFunctions => AzureAppServiceMetadata?.IsRunningMiniAgentInAzureFunctions ?? false;
```

Comment:
What were these settings actually _used_ for? 🤔

Reply:
It was a way to prevent the Serverless Mini Agent from starting for a .NET Windows Azure Function on an App Service Plan since the .NET APM Extension should be used for that case